hazelcast/src/main/java/com/hazelcast/internal/cluster/impl/ClusterServiceImpl.java

```diff
@@ -306,25 +304,49 @@ public void merge(Address newTargetAddress) {
     public void reset() {
         lock.lock();
         try {
-            resetLocalMemberUuid();
+            reset(false);
```

Comment:
minor: `reset()` can be simplified to 
```java
public void reset() {
   reset(false);
}
```
without `lock` and `try/finally` block.